MAINTAINERS.md

```diff
@@ -0,0 +1,36 @@
+# Podman Maintainers
+
+[GOVERNANCE.md](https://github.com/containers/podman/blob/main/GOVERNANCE.md)
+describes the Podman project's governance and the Project Roles used below.
+
+## Maintainers
+
+| Maintainer        | GitHub ID                                                | Project Roles                    | Affiliation                               |
+|-------------------|----------------------------------------------------------|----------------------------------|-------------------------------------------|
+| Brent Baude       | [baude](https://github.com/baude)                        | Core Maintainer                  | [Red Hat](https://www.github.com/redhat/) |
+| Ygal Blum         | [ygalblum](https://github.com/ygalblum)                  | Maintainer                       | [Red Hat](https://www.github.com/redhat/) |
+| Jake Correnti     | [jakecorrenti](https://github.com/jakecorrenti)          | Reviewer                         | [Red Hat](https://www.github.com/redhat/) |
```

Comment:
The Red Hat GH link here is some random user. The proper link is https://github.com/RedHatOfficial

Reply:
Excellent catch, TY